diagnostic.md

```diff
@@ -54,8 +52,7 @@ Using Atom, open up `diagnostic.md` and write your answers below (where indicate
 
 1. Is `/Users/blah_blah/Desktop` a relative path or an absolute path? How do you know?
 
- <!-- Answer Starts Here -->
-
+absolute
```

Comment:
Correct, but it asks you to explain: the first `/` represents the root of the system.